packages/pluggableWidgets/badge-web/src/Badge.editorConfig.ts

```diff
@@ -0,0 +1,31 @@
+import { StructurePreviewProps } from "@widgets-resources/piw-utils";
+import { BadgePreviewProps } from "../typings/BadgeProps";
+
+export function getPreview(values: BadgePreviewProps): StructurePreviewProps {
+    return {
+        type: "RowLayout",
+        columnSize: "grow",
+        children: [
+            {
+                type: "Container",
+                children: [
+                    {
+                        type: "Container",
+                        children: [
+                            {
+                                type: "Text",
+                                content: values.value || "   ",
+                                fontColor: "#FFF",
+                                bold: true
+                            }
+                        ],
+                        padding: values.value ? 9 : 21
```

Comment:
Why don't we follow to 8 pixels rules also here?

Reply:
We are limited in padding capabilities for the moment, because padding is applied to all sides, which made the widget look a bit out of proportion. I have tweaked it now to the 8 grid system (except the empty badge state, because structure preview changes text size when there are just spaces). It looks good, but it can be improved later on to reflect the different padding on different sides better.